library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -150,7 +151,14 @@ public void onCreate() {
 
         downloadClientReadyChecker = getDownloadClientReadyChecker();
 
-        mNotifier = new DownloadNotifier(this, getNotificationImageRetriever(), getResources());
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationDisplayer notificationDisplayer = new NotificationDisplayer(
+                this,
+                notificationManager,
+                getNotificationImageRetriever(),
+                getResources());
+
```

Comment:
This is done here https://github.com/novoda/download-manager/blob/Fix-NPE-removing-stale-notification-tags/library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java#L165 as well and it looks better as well.